src/app/projects/add-edit-project/add-edit-project.component.ts

```diff
@@ -1246,6 +1246,17 @@ export class AddEditProjectComponent implements OnInit, AfterViewInit, OnDestroy
 
   public handleProjectTypesChange(eventData, index) {
     this.myForm.value.projectTypes[index].checked = eventData.checked;
+
+		// If the form's project types have 'Forest Landscape Planning' checked, make sure we display forest project phases.
+		// Select value will be changed to first in new list but form value will be blank, so we need to manually update the form value.
+		const forestTypeIsChecked = this.myForm.value.projectTypes?.[1]?.checked ? true : false;
+		if (forestTypeIsChecked && this.chosenPhases !== Constants.FOREST_PHASES) {
+			this.chosenPhases = Constants.FOREST_PHASES;
+			this.myForm.controls.projectPhase.setValue('Pre-Planning');
```

Comment:
It's not totally clear from the PO's description in the ticket, but I think he said he wanted the dropdown to go blank here. Would he be alright if it pre-populates with the first phase?

Reply:
My interpretation of the PO request is that the value is blank on page load if the stored value does not match one of the options in the list. When toggling between project types, the angular select box does not want to revert to an unset state, even if the form value reverts to unset. It will only accept a change if it's within the list of values. If you think it's a problem, I can try to revert the toggle to an unset state. However, I don't think that's what Steve is actually asking for.